rts/Game/GameHelper.cpp

```diff
@@ -934,10 +934,13 @@ void CGameHelper::BuggerOffRectangle(const float3& mins, const float3& maxs, boo
 	const int allyTeamId = teamHandler.AllyTeam(teamId);
 
 	for (CUnit* u : *qfQuery.units) {
-		if (u->moveDef == nullptr) { continue; }
+		if (u->unitDef == nullptr || u->unitDef->IsImmobileUnit()) { continue; }
```

Comment:
Can `u->unitDef` ever be null?

Reply:
That's probably me being to defensive in my coding. Removed.